UDP/course_materialized_view.md

```diff
@@ -122,3 +92,15 @@ REFRESH MATERIALIZED view v1_student_current_term_course;
 REFRESH MATERIALIZED view v2_student_current_term_course_assignment_avg;
 REFRESH MATERIALIZED view v3_student_current_term_course_activities;
 ```
+
+## Auto-Refreshed Materialized Views
+Details are here: https://resources.unizin.org/display/UDP/UDP+Context+store
+You can adjust the query at the bottom of the resources page link to verify all views have the right owner attached:
+
```

Comment:
Since the refresh details are in the linked UDP page, there is need to repeat the query here.

Reply:
OK, I removed the query and keeping only the link

Reply:
Great! Thanks!